test/test_utilities.cpp

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright 2017-2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "test_utilities.h"
+
+using namespace std;
+
+namespace ng = ngraph;
+
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+void ActivateNGraph() {
+  setenv("NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS", "1", 1);
+  unsetenv("NGRAPH_TF_DISABLE");
+}
+
+void DeactivateNGraph() {
+  unsetenv("NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS");
+  setenv("NGRAPH_TF_DISABLE", "1", 1);
+}
+
+void AssertTensorEquals(Tensor& T1, Tensor& T2) {
+  auto T_size = T1.flat<float>().size();
+  auto T1_data = T1.flat<float>().data();
```

Comment:
We should also add an assert on the shape here.

Reply:
Yes, most review comments from crucible have not been addressed yet. Definitely in the next PR.